app/main.js

```diff
@@ -89,16 +89,41 @@ function startPowerMonitoring () {
   })
 }
 
-function displaysX (width = 800) {
+function numDisplays () {
```

Comment:
maybe rename this to something more meaningful? Like `displaysSize`, `numberOfDispalys`?

Reply:
Agreed, `numberOfDisplays` is a good replacement!